com.unity.template.renderstreaming-hd/Assets/Scripts/Signaling/FurioosSignaling.cs

```diff
@@ -57,7 +57,10 @@ public void Stop()
 
         public event OnSignedInHandler OnSignedIn;
         public event OnOfferHandler OnOffer;
+        #pragma warning disable 0067
+        // this event is never used in this class
         public event OnAnswerHandler OnAnswer;
```

Comment:
If this event is not used in this class, then why do we need to declare it here ?

Reply:
I believe this event is needed to implement interface class.
Is it right? @kannan-xiao4

Reply:
Yes, that's right.
This event definition use for define interface about receive answer.